src/code/speed_meter.c

```diff
@@ -1,25 +1,60 @@
 #include "global.h"
 #include "vt.h"
 
-volatile OSTime D_8016A520;
-volatile OSTime D_8016A528;
-volatile OSTime D_8016A530;
-volatile OSTime D_8016A538;
-volatile OSTime D_8016A540;
-volatile OSTime D_8016A548;
-volatile OSTime D_8016A550;
-volatile OSTime D_8016A558;
-volatile OSTime gRSPAudioTotalTime;
-volatile OSTime gRSPGFXTotalTime;
-volatile OSTime gRSPOtherTotalTime;
+/**
+ * How much time the audio update on the audio thread (`func_800E4FE0`) took in total, between scheduling the last two
+ * graphics tasks.
+ */
+volatile OSTime gAudioThreadUpdateTimeTotalPerGfxTask;
+/**
+ * How much time elapsed between scheduling the previous graphics task and the current one being ready (the previous
+ * task not necessarily being finished yet), without the amount of time spent on the audio update in the audio thread.
+ */
+volatile OSTime gGfxTaskSentToNextReadyMinusAudioThreadUpdateTime;
+/**
+ * How much time the RSP ran audio tasks for over the course of `gGraphUpdatePeriod`.
+ */
+volatile OSTime gRSPAudioTimeTotal;
+/**
+ * How much time the RSP ran graphics tasks for over the course of `gGraphUpdatePeriod`.
+ * Typically the RSP runs 1 graphics task per `Graph_Update` cycle, but may run 0 (see `Graph_Update`).
+ */
+volatile OSTime gRSPGfxTimeTotal;
+/**
+ * How much time the RDP ran for over the course of `gGraphUpdatePeriod`.
+ */
+volatile OSTime gRDPTimeTotal;
+/**
+ * How much time elapsed between the last two `Graph_Update` ending.
+ * This is expected to be at least the duration of a single frame, since it includes the time spent waiting on the
+ * graphics task to be done.
+ */
+volatile OSTime gGraphUpdatePeriod;
+/**
+ * The time at which the audio thread audio update started.
+ */
+volatile OSTime gAudioThreadUpdateTimeStart;
+
+// Accumulator for `gAudioThreadUpdateTimeStart`
+volatile OSTime gAudioThreadUpdateTimeAcc;
+// Accumulator for `gRSPAudioTimeTotal`
+volatile OSTime gRSPAudioTimeAcc;
+// Accumulator for `gRSPGfxTimeTotal`.
+volatile OSTime gRSPGfxTimeAcc;
+volatile OSTime gRSPOtherTimeAcc;
 volatile OSTime D_8016A578;
-volatile OSTime gRDPTotalTime;
+// Accumulator for `gRDPTimeTotal`
+volatile OSTime gRDPTimeAcc;
```

Comment:
I would put some line breaks through all of this just to make it clear what comment belongs to what
(same comment for the 2 variables with block comments in graph.c)